nrt/data/IEEE14/IEEE14_WithAutomata/IEEE14_DistanceProtection/IEEE14.dyd

```diff
@@ -0,0 +1,123 @@
+    <dyn:macroStaticRef id="LOAD"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="_LOAD___4_EC" lib="LoadAlphaBeta" parFile="IEEE14.par" parId="1" staticId="_LOAD___4_EC">
+    <dyn:macroStaticRef id="LOAD"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="_LOAD___5_EC" lib="LoadAlphaBeta" parFile="IEEE14.par" parId="1" staticId="_LOAD___5_EC">
+    <dyn:macroStaticRef id="LOAD"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="_LOAD___6_EC" lib="LoadAlphaBeta" parFile="IEEE14.par" parId="1" staticId="_LOAD___6_EC">
+    <dyn:macroStaticRef id="LOAD"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="_LOAD___9_EC" lib="LoadAlphaBeta" parFile="IEEE14.par" parId="1" staticId="_LOAD___9_EC">
+    <dyn:macroStaticRef id="LOAD"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="_LOAD__10_EC" lib="LoadAlphaBeta" parFile="IEEE14.par" parId="1" staticId="_LOAD__10_EC">
+    <dyn:macroStaticRef id="LOAD"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="_LOAD__11_EC" lib="LoadAlphaBeta" parFile="IEEE14.par" parId="1" staticId="_LOAD__11_EC">
+    <dyn:macroStaticRef id="LOAD"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="_LOAD__12_EC" lib="LoadAlphaBeta" parFile="IEEE14.par" parId="1" staticId="_LOAD__12_EC">
+    <dyn:macroStaticRef id="LOAD"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="_LOAD__13_EC" lib="LoadAlphaBeta" parFile="IEEE14.par" parId="1" staticId="_LOAD__13_EC">
+    <dyn:macroStaticRef id="LOAD"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="_LOAD__14_EC" lib="LoadAlphaBeta" parFile="IEEE14.par" parId="1" staticId="_LOAD__14_EC">
+    <dyn:macroStaticRef id="LOAD"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="GEN____1_SM" lib="GeneratorSynchronousFourWindingsProportionalRegulations" parFile="IEEE14.par" parId="2" staticId="_GEN____1_SM">
+    <dyn:macroStaticRef id="GEN"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="GEN____2_SM" lib="GeneratorSynchronousFourWindingsProportionalRegulations" parFile="IEEE14.par" parId="3" staticId="_GEN____2_SM">
+    <dyn:macroStaticRef id="GEN"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="GEN____3_SM" lib="GeneratorSynchronousFourWindingsProportionalRegulations" parFile="IEEE14.par" parId="4" staticId="_GEN____3_SM">
+    <dyn:macroStaticRef id="GEN"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="GEN____6_SM" lib="GeneratorSynchronousThreeWindingsProportionalRegulations" parFile="IEEE14.par" parId="5" staticId="_GEN____6_SM">
+    <dyn:macroStaticRef id="GEN"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="GEN____8_SM" lib="GeneratorSynchronousThreeWindingsProportionalRegulations" parFile="IEEE14.par" parId="6" staticId="_GEN____8_SM">
+    <dyn:macroStaticRef id="GEN"/>
+  </dyn:blackBoxModel>
+  <dyn:blackBoxModel id="OMEGA_REF" lib="DYNModelOmegaRef" parFile="IEEE14.par" parId="7"/>
+  <dyn:blackBoxModel id="DISCONNECT_LINE" lib="EventQuadripoleDisconnection" parFile="IEEE14.par" parId="9"/>
```

Comment:
if DISCONNECT_LINE is not used, can we remove it here and in the .par?

Same in all the nrt